extensions/iceberg/s3/src/main/java/io/deephaven/iceberg/util/AsyncHttpClientProperties.java

```diff
@@ -0,0 +1,93 @@
+//
+// Copyright (c) 2016-2025 Deephaven Data Labs and Patent Pending
+//
+package io.deephaven.iceberg.util;
+
+import java.time.Duration;
+import java.util.Map;
+import org.apache.iceberg.util.PropertyUtil;
+import org.jetbrains.annotations.NotNull;
+import software.amazon.awssdk.http.crt.AwsCrtAsyncHttpClient;
+import software.amazon.awssdk.http.nio.netty.NettyNioAsyncHttpClient;
+import software.amazon.awssdk.services.s3.S3AsyncClientBuilder;
+
+class AsyncHttpClientProperties {
+
+    // Add option for CRT and inject this property
+    public static final String CLIENT_TYPE = "http-client.async.client-type";
```

Comment:
I think there should be a unique prefix that doesn't overlap with `org.apache.iceberg.aws.HttpClientProperties#CLIENT_PREFIX` (`http-client.`); maybe, `http-client-async.`?